spotbugs/etc/messages.xml

```diff
@@ -1819,6 +1819,23 @@ factory pattern to create these objects.</p>
       ]]>
     </Details>
   </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.FindHidingSubClass">
+    <Details>
+      <![CDATA[
+            <p>
+              Hiding happens when a sub class defines a static method
+              with same header (signature plus return type) as any of its super classes.
+              In case of method hiding, the invoked method is determined based on the specific qualified name or
+              method invocation expression used at the calling site.
+              The results are often unexpected, although the Java language provides unambiguous rules for method invocation for method hiding.
+              Moreover, method hiding and method overriding is often confused by programmers.
+              Consequently, programmers must avoid the method hiding.
```

Comment:
I think this is a "should" rather than "must".

Reply:
Fixed.